website-1.x/blog/2021-05-11-releasing-docusaurus-beta.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Releasing Docusaurus 2 beta
+author: Sébastien Lorber
+authorTitle: Docusaurus maintainer
+authorURL: https://sebastienlorber.com
+authorImageURL: https://github.com/slorber.png
+authorTwitter: sebastienlorber
+tags: [release, beta]
+# image: /img/blog/2021-03-09-releasing-docusaurus-i18n/social-card.png
+---
+
+Today, we officially release the first **Docusaurus 2 beta**, after a **very long alpha period**.
+
+We believe Docusaurus 2 is ready for **mainstream adoption**!
+
+<!--truncate-->
+
+## Docusaurus adoption
+
+**Don't be afraid by the beta tag!**
+
+Docusaurus 2 is **widely adopted** and growing fast:
+
+![Docusaurus growth](/img/blog/2021-05-11-releasing-docusaurus-beta/trend.png)
+
+Our new [showcase](https://docusaurus.io/showcase) page allow to **filter sites by features**: get inspired by production-ready sites!
+
+Don't miss our [favorite](https://docusaurus.io/showcase?tags=favorite) sites, they all **stand out** with something unique:
+
+[![Docusaurus growth](/img/blog/2021-05-11-releasing-docusaurus-beta/favorites.png)](https://docusaurus.io/showcase?tags=favorite)
+
+## Why being in alpha for so long?
+
+The first alpha release [v2.0.0-alpha.0](https://github.com/facebook/docusaurus/releases/tag/v2.0.0-alpha.0) was **2 years ago** 😅.
+
+It is definitively a very long and unusual alpha period.
+
+Docusaurus 2 has reached **feature parity** with Docusaurus 1 with **[i18n](https://docusaurus.io/blog/2021/03/09/releasing-docusaurus-i18n)**, and it has been **successfully adopted** by many Docusaurus sites (see [i18n showcase](https://docusaurus.io/showcase?tags=i18n)).
+
+We can now confidently say that the **core features** of Docusaurus 2 are **stable enough** to be in beta.
+
+## What does being in beta mean?
+
+We will use the beta phase to **improve our theming system**.
+
+We want to make it:
+
+- **easier to implement a custom theme**, including for ourselves. We want to provide [multiple official themes](https://github.com/facebook/docusaurus/issues/3522) (including [Tailwind](https://github.com/facebook/docusaurus/issues/2961)) for a long time.
+
+- **safer to extend an existing theme**: it is sometimes painful to upgrade a **highly customized** Docusaurus site, as customizations can **conflict with internal changes**. We need to make the theme public API surface more explicit, and make it clear what is safe to customize.
```

Comment:
```suggestion
- **safer to extend an existing theme**: it can be painful to upgrade a **highly customized** Docusaurus site, as customizations can **conflict with internal changes**. We need to make the theme public API surface more explicit, and make it clear what is safe to customize.
```